interface/patient_file/merge_patients.php

```diff
@@ -93,6 +93,58 @@ function updateRows($tblname, $colname, $source_pid, $target_pid)
     }
 }
 
+function mergeRows($tblname, $colname, $source_pid, $target_pid)
+{
+    global $PRODUCTION;
+    $crow = sqlQuery("SELECT COUNT(*) AS count FROM " . escape_table_name($tblname) . " WHERE " . escape_sql_column_name($colname, array($tblname)) . " = ?", array($source_pid));
+    $count = $crow['count'];
+    if ($count) {
+        echo "<br />lists_touch count is ($count)";
+        $source_array = array();
+        $source_sel = "SELECT * FROM " . $tblname . " WHERE `pid` = ?";
+        $source_res = sqlStatement($source_sel, array($source_pid));
+
+        $target_array = array();
+        $target_sel = "SELECT * FROM " . escape_table_name($tblname) . " WHERE `pid` = ?";
+        $target_res = sqlStatement($target_sel, array($target_pid));
+
+        while ($source_row = sqlFetchArray($source_res)) {
+            while ($target_row = sqlFetchArray($target_res)) {
+                if ($source_row['type'] == $target_row['type']) {
+                    if (strcmp($target_row['date'], $source_row['date']) < 0) {
+                        // we delete the entry from the target since the source has a newer date
```

Comment:
I gave wrong advice on call. We should keep the earliest (rather than the latest) entry :)
(this is because lists_touch is done only once, so should keep the first one: https://github.com/openemr/openemr/blob/master/library/lists.inc#L134-L144)